chainer/cuda.py

```diff
@@ -237,16 +237,17 @@ def to_gpu(array, device=None, stream=None):
     """Copies the given CPU array to the specified device.
 
     Args:
-        array: Array to be sent to GPU.
+        array (numpy.ndarray, cupy.ndarray, list or tuple): Array or arrays to
```

Comment:
Do we need to write that `None` is supported?

Reply:
Fixed.